src/main/resources/pdl/pdl-api-schema.graphql

```diff
@@ -47,8 +59,7 @@ type Endring {
     """
 
     Opphavet til informasjonen. I NAV blir dette satt i forbindelse med registrering (f.eks: Sykehuskassan).
-    Fra Folkeregisteret fÃ¥r vi opphaven til dems opplysning, altsÃ¥ NAV, UDI, Politiet, Skatteetaten o.l.. Fra Folkeregisteret
-    kan det ogsÃ¥ vÃ¦re tekniske navn som: DSF_MIGRERING, m.m..
+    Fra Folkeregisteret fÃ¥r vi opphaven til dems opplysning, altsÃ¥ NAV, UDI, Politiet, Skatteetaten o.l.. Fra Folkeregisteret kan det ogsÃ¥ vÃ¦re tekniske navn som: DSF_MIGRERING, m.m..
```

Comment:
Kan vel kanskje fikse de norske bokstavene her? Er vel bare copy-paste som har dratt med seg dette?

Reply:
pdl-api-schema genereres ved bruk av intellij-graphql-pluginen.

Jeg stemmer egentlig for å slette denne, jeg tror ikke den har noe i kodebasen å gjøre.